5c-keyof-typeof-ex1.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { strict as assert } from "assert";
+
+import { getDinoFact, dinosaurFacts } from "./5c-keyof-typeof-ex1";
+
+describe("Keyof Typeof Exercise 1", () => {
+  it("gets dino facts", () => {
+    assert.deepEqual(
+      getDinoFact(dinosaurFacts, "t-rex"),
+      dinosaurFacts["t-rex"]
```

Comment:
These pass without any changes necessary since the types are set to `any`. I could prevent that from using something like `never` or `unknown` would that be preferred?